Source/KakapoServer.swift

```diff
@@ -37,15 +39,20 @@ import Foundation
  ```
  */
 public final class KakapoServer: NSURLProtocol {
-    
+
+    /**
+     The overall list of `Router` objects, which is know to all `KakapoServer` instances.
+     Visibility is on the `KakapoServer` class level, because `NSURLProtocol` just allows us to register
+     class elements (no instances are possible).
+     */
```

Comment:
Nothing special needed, private prop/methods/types are not accessible anyway. However we try to limit comments , only when really needed